### ISSUE TITLE
Need help with running spotbugs in CLI mode

### ISSUE THREAD

**Issue by rohitCloudDefense** [spotbugs/spotbugs#2048]
I was able to run SpotBugs with gui using below command

`sh /Users/rohitcoder/Downloads/spotbugs/bin/spotbugs`

I tried running this in CLI mode, did lot of research still confused. I ran below command, can someone please guide me whats the issue, why it's not scanning ?

`sh /Users/rohitcoder/Downloads/spotbugs/bin/spotbugs -textui -output ~/Desktop/spotbugs.txt -xml -low -high -project ~/Desktop/CloudDefense/redisson/`

**Comment by KengoTODA** [comment]
Several problems I found:

1. The [`-project` option](https://spotbugs.readthedocs.io/en/stable/running.html#common-command-line-options) should point to a `.fb` file, not a directory to scan.
2. You specify command-line options, but you use the [wrapper script](https://spotbugs.readthedocs.io/en/stable/running.html#wrapper-script-command-line-options) that does not accept them.

I suggest you [invoke `spotbugs.jar` directly](https://spotbugs.readthedocs.io/en/stable/running.html#direct-invocation-of-spotbugs) instead of the wrapper script. In my understanding, the script is not so maintained after the forking from findbugs. It will be like the below:

```console
$ java -jar path/to/spotbugs.jar -textui -xml:spotbugs.xml -low ~/Desktop/CloudDefense/redisson/
```

**Comment by rohitCloudDefense** [comment]
Hi @KengoTODA 

Thanks for the help! The code which you provide was having some typo i think, it was throwing same output as previous with an error message `unknown option: -xml:spotbugs.xml` 

<img width="1205" alt="Screenshot 2022-05-13 at 9 14 32 AM" src="https://user-images.githubusercontent.com/104613917/168207139-4b8d0764-e657-4865-9622-397dfcc4896f.png">

I fixed this by changing `:`with `=` in command and now it's working perfectly

```bash
java -jar /Users/rohitcoder/Downloads/spotbugs/lib/spotbugs.jar -textui -xml=spotbug.xml -low ~/Desktop/CloudDefense/redisson/
```

This above code will help someone in future, if they face same issue.

**Comment by KengoTODA** [comment]
seems that your SpotBugs is a little bit old, better to use the latest release.

**Comment by rohitCloudDefense** [comment]
Using latest version 4.7.0

**Comment by KengoTODA** [comment]
Aha, I understood, SpotBugs user manual is incorrect and I will fix it by #2051. Thanks!
